OpenRA.Mods.Common/Traits/Render/WithAcceptDeliveredCashAnimation.cs

```diff
@@ -0,0 +1,68 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits.Render
+{
+	[Desc("Replaces the building animation when it accepts a cash delivery unit.")]
+	public class WithAcceptDeliveredCashAnimationInfo : ITraitInfo, Requires<WithSpriteBodyInfo>
+	{
+		[Desc("Sequence name to use")]
+		[SequenceReference] public readonly string Sequence = "active";
+
+		public readonly bool PauseOnLowPower = false;
```

Comment:
Could you please make this a `PausableConditionalTrait` and use `IsTraitPaused` instead?

This would become a must after #13998 is merged anyway, at least this way we save ourselves the hassle of needing an upgrade rule for this trait.